pds4indextools/template_pds.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe call this file `index_label_template_pds.xml` to make it clear it's for the index labels?